jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarProcessorTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.jar;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.tools.jib.cli.jar.JarProcessor.JarType;
+import com.google.common.io.Resources;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.Test;
+
+public class JarProcessorTest {
+
+  private static final String SPRINGBOOT_RESOURCE_DIR = "jar/springboot";
```

Comment:
```suggestion
  private static final String SPRING_BOOT_RESOURCE_DIR = "jar/springboot";
```
And maybe this can just be `jar/springboot/springbootJar.jar`?